filebeat/filebeat.yml

```diff
@@ -142,12 +142,18 @@ setup.kibana:
 
 #================================ Outputs =====================================
 
+# Console
```

Comment:
This should not be commited. You need to run `make update` and push again.

Reply:
Seems I have gotten my local repo a bit out of sync and having trouble pushing it. I've tried to resolve them with manual updates, but let me know if you think I should just put my changes into a freshly branched PR off of the current master.